server/utils/tasks/listTable.ts

```diff
@@ -89,9 +91,22 @@ const allocationCell = (task: Task): TableCell => ({
   text: task.allocatedToStaffMember?.name,
 })
 
+const getPersonName = (personSummary: PersonSummary): string => {
+  switch (personSummary.personType) {
+    case 'FullPersonSummary':
+      return (personSummary as FullPersonSummary).name
+    case 'RestrictedPersonSummary':
+      return `LRN CRN: ${personSummary.crn}`
```

Comment:
Change `LRN` to `LAO` please

Reply:
@Rosie-Brigham pushed the changes please check again